sympy/assumptions/tests/test_refine.py

```diff
@@ -144,6 +145,12 @@ def test_atan2():
     assert refine(atan2(y, x), Q.negative(y) & Q.zero(x)) == -pi/2
     assert refine(atan2(y, x), Q.zero(y) & Q.zero(x)) == nan
 
+def test_matrixelement():
```

Comment:
`git revert` all the matrix related changes. Learn about git branch as well.

- Always start new PR with new branch -- checkout from master branch only.
- Here you can either **revert** all the matrix related commits and force push.

Reply:
Yes. I'll take care the next time.